docs/index.md

```diff
@@ -1,8 +1,11 @@
 # Overview
 
-Codelingo is a Platform as a Service (PaaS) for software development teams to solve software development problems. It treats your software as data and automates your workflows, called Actions, with the rules and patterns you define, called Rules.
+Codelingo is a platform for software development teams to store, discuss and
+code their tacit knowledge into project specific rules.
```

Comment:
```suggestion
CodeLingo is a platform for software development teams to store, discuss, and
codify their tacit knowledge into project specific rules.
```

Reply:
Thoughts on 'rules' vs 'Rules'? I feel like capitalising our own nomenclature helps to highlight their significance in sentences. Also, up to you on the Oxford comma ;)

Reply:
Okay, after reading more, it seems you _are_ capitalising. There's just a few that you haven't. Can you please make those all consistent?